### ISSUE TITLE
WCAG 4.1.2: Ensure aria-hidden elements are not focusable nor contain focusable elements (.group\/sidebar-wrapper)

### ISSUE THREAD

**Issue by azap026** [azap026/smetalabv3#126]
#### Issue
`Ensure aria-hidden elements are not focusable nor contain focusable elements` ([aria-hidden-focus](https://accessibilityinsights.io/info-examples/web/aria-hidden-focus))
#### Target application
[Smetalab - Smart Engineering & Management](http://localhost:3000/app)
#### Element path
.group\/sidebar-wrapper
#### Snippet
`<div data-slot="sidebar-wrapper" style="--sidebar-width:16rem;--sidebar-width-icon:3rem" class="group/sidebar-wrapper has-data-[variant=inset]:bg-sidebar flex min-h-dvh w-full" data-aria-hidden="true" aria-hidden="true">`
#### How to fix
    Fix all of the following:
      Focusable content should be disabled or be removed from the DOM
      Focusable content should have tabindex="-1" or be removed from the DOM
#### Environment
Chrome version 144.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
